crates/polars-lazy/src/tests/streaming.rs

```diff
@@ -63,10 +63,7 @@ fn test_streaming_csv() -> PolarsResult<()> {
 
 #[test]
 fn test_streaming_glob() -> PolarsResult<()> {
-    let q = get_csv_glob();
-    let q = q.sort(["sugars_g"], Default::default());
```

Comment:
I'm not sure how this was passing before, but there was sort instability for this query